### PR TITLE
[visual][popwin] add keybiding to resume last popwin buffer

### PR DESCRIPTION
**Pull request by thanhvg**
bind it to `spc r w`

### CODE REVIEW

**Comment by Kazark** [comment]
I noticed that some Helm searches can be resumed with `SPC s l` but not always. Is this related at all? I would really love to be able to resume all Helm searches.

**Comment by thanhvg** [comment]
No this just resumes popwin buffer such as *help* buffer,  lsp document at point buffer etc. 

Re helm I don't know, helm seems to prevents some commands from resuming I switched to ivy and it seems that ivy can resume almost anything. Btw I also bind `spc r L` to `C-u spc r l` not sure if it's worth a PR but it is extremely useful.

File: layers/+spacemacs/spacemacs-visual/config.el [hunk 1]
```diff
@@ -0,0 +1,13 @@
+;;; config.el --- Spacemacs UI Visual Layer packages File
+;;
+;; Copyright (c) 2012-2021 Sylvain Benner & Contributors
+;;
+;; Author: Sylvain Benner <sylvain.benner@gmail.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
```
Comment (smile13241324) [inline layers/+spacemacs/spacemacs-visual/config.el:10]: This line must be replaced with the proper GPLv3 header

**Review by smile13241324 — CHANGES_REQUESTED** [review]
Nice change, only missing thing is the GPLv3 header we have been asked to add, otherwise we will get into trouble with the FSF.

**Comment by smile13241324** [comment]
Yeah helm is a buggy beast, still I largly prefer it to ivy which I find so much lacking in comparison.

**Comment by Kazark** [comment]
@smile13241324 off the top of my head, certainly I can't resume `SPC s s`. I'll post the other ones here as a re-encounter them.

**Review by smile13241324 — APPROVED** [review]
:+1:

**Comment by smile13241324** [comment]
Thank you for contributing to spacemacs 💜, I have cherry picked your commit into develop.

**Comment by smile13241324** [comment]
Interesting, never noticed that there is a difference between the two. Thanks for the hint I will double check

**Pull request closed without merging**